addons/web/models/res_users_settings.py

```diff
@@ -13,23 +14,34 @@ def _format_settings(self, fields_to_format):
             res['embedded_actions_config_ids'] = self.embedded_actions_config_ids._embedded_action_settings_format()
         return res
 
-    def get_embedded_actions_settings(self):
+    def get_embedded_actions_settings(self, action_id=None, res_model=None, res_ids=None):
```

Comment:
Was it necessary to complexify the API? I understand that you didn't want to reload the whole embedded actions settings, but I don't think it would be a problem to do so, and the code would be simpler.

Reply:
My point is that we could consider reloading the complete embedded actions settings (same as what we do in session_info). Why would it be an issue?